pkg/release/push.go

```diff
@@ -374,3 +371,16 @@ func (p *PushBuild) copyStageFiles(stageDir string, files []stageFile) error {
 
 	return nil
 }
+
+// PushReleaseArtifacts can be used to push local artifacts from the `srcPath`
+// to the remote `gcsPath`. The Bucket has to be set via the `Bucket` option.
+func (p *PushBuild) PushReleaseArtifacts(srcPath, gcsPath string) error {
+	logrus.Info("Pushing release artifacts")
+
+	copyOpts := gcs.DefaultGCSCopyOptions
+	copyOpts.NoClobber = pointer.BoolPtr(p.opts.AllowDup)
+
+	return errors.Wrap(gcs.CopyToGCS(
+		srcPath, filepath.Join(p.opts.Bucket, gcsPath), copyOpts,
+	), "copy artifacts to GCS")
```

Comment:
:astonished: oooh no if `err != nil` !!!

Reply:
Do you think it’s misleading? It will not produce an error (wrap) if err == nil.

Reply:
I think it is nice as it saves some repetition it was more surprising than misleading. I actually went to check in the errors.Wrap() sources to verify :)

At first glance it looks a but like it is returning an error, but otherwise its nice to save some lines.